packages/openapi3/test/openapi-output.test.ts

```diff
@@ -160,6 +160,19 @@ describe("openapi3: operations", () => {
     strictEqual(getThing.parameters[1].schema.minimum, 1);
     strictEqual(getThing.parameters[1].schema.maximum, 10);
   });
+
+  it("deprecate operations with @deprecated", async () => {
+    const res = await openApiFor(
+      `
+      @deprecated("use something else")
+      @route("/")
```

Comment:
Would just remove the `@route` and `@get` decorators for the test. They don't add much value but make the test longer

Reply:
good point.